README.md

```diff
@@ -66,6 +66,7 @@ See [Testing Farm docs](https://docs.testing-farm.io) for more information on su
 | `update_pull_request_status` | Action will update pull request status. Default: true | true |
 | `environment_settings` | Pass custom settings to the test environment. Default: {} | empty |
 | `pr_head_sha` | SHA of the latest commit in PR. Used for communication with GitHub API. | $(git rev-parse HEAD) |
+| `create_github_summary` | Create summary of the Testing Farm as GiHub Action job. | false |
```

Comment:
I think this could be true. Or even we can create it always.

Reply:
Sure, I see no problem in setting this as permanent, if you think that would be more useful.

Reply:
As I think about it further, I do not think that hardcoding this is a good idea. It is possible that someone would like to have different summary, possibly containing data from other actions too.

I would revert the change if you agree.

Reply:
Fixed in [edd265e](https://github.com/sclorg/testing-farm-as-github-action/pull/74/commits/edd265e3bcc41dc40c07941c7eb86ac0ce40f3b3).